### PR TITLE
introducing gitter

### PR DESCRIPTION
**Pull request by ndzoesch**

### CODE REVIEW

File: source/blog/_posts/2017-09-29-gitter.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- dev
+
+authors: [nd]
+github_link: blog/_posts/2017-09-29-gitter.md
+
+---
+
+## A huge change for the community
+
+For several years now, since 2011 to be precise, there is a small but vibrant chat community on freendode in the #shopware channel.
```
Comment (janbuecker) [inline source/blog/_posts/2017-09-29-gitter.md]: freenode

File: source/blog/_posts/2017-09-29-gitter.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+All these people chatting there have one thing in common, they are working on the shopware ecommerce system, in one way or the other.
```
Comment (janbuecker) [inline source/blog/_posts/2017-09-29-gitter.md]: Shopware ist ein Eigenname

File: source/blog/_posts/2017-09-29-gitter.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Now these times will be over. As of the end of October this year, the IRC channel will be closed.
```
Comment (janbuecker) [inline source/blog/_posts/2017-09-29-gitter.md]: be precise: October 2017

File: source/blog/_posts/2017-09-29-gitter.md (hunk 1 above)
Comment (mitelg) [reply source/blog/_posts/2017-09-29-gitter.md]: hahahaa lol :laughing:

File: source/blog/_posts/2017-09-29-gitter.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+So, why the change? What changes? What are the consequences? Just read further, friend, and find out.
+
+## What will change?
+
+For a short period of time (read: one month) the IRC channel will stay open. During this period we will actively transfer people to [Gitter](https://gitter.im/shopware/shopware).
+
+After this, the IRC channel will be closed and all chat activity will take place on Gitter.
+
+If you are an IRC user by now, you can use your client for Gitter as well, just follow the steps mentioned at the end of the [contribution page](/contributing/gitter).
```
Comment (janbuecker) [inline source/blog/_posts/2017-09-29-gitter.md]: {{ site.url }}/contributing/gitter

File: source/blog/_posts/2017-09-29-gitter.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+Like IRC, but with more features and more or less integrated with our github repository. Like Slack, but more community-centric.
+
+If you want to know the full story, there is a nice [wikipedia article](https://en.wikipedia.org/wiki/Gitter) about the service.
+
+*One of the biggest changes for former IRC users:* Gitter chatrooms are public and save history indefinitely. So everything you say is wisdom for eternity.
+Or embarrassment, but this is an unusual state of mind for shopware enthusiasts. So be careful. There be dragons. On the other hand, you don't need that icky [bouncer](https://en.wikipedia.org/wiki/BNC_(software)) anymore. ;)
+
+## But... why?
+
+To cite the devdocs [irc page](/contributing/irc):
```
Comment (janbuecker) [inline source/blog/_posts/2017-09-29-gitter.md]: please check all links and include {{ site.url }}

File: source/blog/_posts/2017-09-29-gitter.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+## A huge change for the community
+
+For several years now, since 2011 to be precise, there is a small but vibrant chat community on freenode in the #shopware channel.
+
+All these people chatting there have one thing in common, they are working on the Shopware ecommerce system, in one way or the other.
+
+Now these times will be over. As of the end of October 2017, the IRC channel will be closed.
+If you hear a faint scream in the distance it may be one of the guys from the IRC channel who is just reading this post as you do now.
```
Comment (bcremer) [inline source/blog/_posts/2017-09-29-gitter.md]: /me screeeeeeeeeeeeeeeeeeeeams!

**Pull request merged**